README.md

```diff
@@ -1,7 +1,29 @@
-# Recoil
+# Recoil &middot; [![Node.js CI](https://github.com/facebookexperimental/Recoil/workflows/Node.js%20CI/badge.svg)](https://github.com/facebookexperimental/Recoil/actions) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/facebookexperimental/Recoil/blob/master/LICENSE)
 
-[![Node.js CI](https://github.com/facebookexperimental/Recoil/workflows/Node.js%20CI/badge.svg)](https://github.com/facebookexperimental/Recoil/actions)
+Recoil is an experimental set of utilities for state management with React.<br>
```

Comment:
shouldn't need to include `<br>` tags in markdown. Can just leave an empty line between.